### PR TITLE
fix(readme): poor man PSC indicator badge

### PR DESCRIPTION
**Pull request by yajo**
Provide a more ergonomic way for https://github.com/OCA/oca-addons-repo-template/issues/140#issuecomment-1233973128 to fix https://github.com/OCA/oca-addons-repo-template/issues/140.

### CODE REVIEW

**Review by carmenbianca — APPROVED** [review]
LGTM, even if it's not ideal.

Two suggestions:

- Keep an issue open somewhere to tag this as in need of improvement.
- Consider using DuckDuckGo instead of Google :)

File: src/README.md.jinja
Comment (legalsylvain) [inline src/README.md.jinja:6]:
```suggestion
[![Find PSC team](https://img.shields.io/badge/PSC-Find%20team-875A7B.png)](https://duckduckgo.com/?q=site%3Ahttps%3A%2F%2Fodoo-community.org%2Fpsc-teams+%22{{ repo_slug | urlencode }}%22)
```

this should work @carmenbianca

**Comment by sbidoul** [comment]
We should rather link to https://oca.github.io/repo-maintainer-conf/, because https://odoo-community.org/psc-teams.

With some tweaks to https://github.com/OCA/repo-maintainer-conf/blob/master/.github/workflows/static.yml and the script it invokes, we should even be able to have a rich man solution.

@simahawk @etobella are we ready to remove https://odoo-community.org/psc-teams or replace it with redirects to ttps://oca.github.io/repo-maintainer-conf ?

File: src/README.md.jinja
Comment (yajo) [reply src/README.md.jinja:6]: Sadly, not:

image

**Comment by yajo** [comment]
> We should rather link to https://oca.github.io/repo-maintainer-conf/, because https://odoo-community.org/psc-teams is unmaintained.

Sadly, we can't either:
image

**Comment by yajo** [comment]
> With some tweaks to https://github.com/OCA/repo-maintainer-conf/blob/master/.github/workflows/static.yml and the script it invokes, we should even be able to have a rich man solution.

Do you think it's worth waiting for this to happen? Or could we merge this workaround and just update the badge url once the rich man solution lands?

**Comment by legalsylvain** [comment]
Do you think it's worth waiting for this to happen?

As it is changing the template of OCA modules, better to not push a temporary solution IMO that will be replaced by another in a few time. Matter of taste, though.

**Pull request closed without merging**
